src/LSST/control/ControlMain.h

```diff
@@ -47,20 +56,52 @@ class ControlMain {
     /// Verify threads are stopped and joined.
     ~ControlMain();
 
-    /// Return a reference to the global ControlMain instance.
-    /// @throws `ConfigException` if `setup` has not already been called.
-    static ControlMain& get();
-
     /// Return a shared pointer to the global ControlMain instance.
     /// @throws `ConfigException` if `setup` has not already been called.
     static Ptr getPtr();
 
+    /// &&& doc
+    void run(int argc, const char* argv[]);
+
+    /// Returns true once the server is running.
+    bool getRunning() { return _running; }
```

Comment:
`isRunning()` might be a better name for the method.

Reply:
agreed